### PR TITLE
(chore)docs: update getting started doc with analytics note

### PR DESCRIPTION
**Pull request by ksatchit**
- Add note for analytics in getstarted page

Signed-off-by: ksatchit <karthik.s@mayadata.io>

### CODE REVIEW

**Review by chandankumar4 — APPROVED** [review]

File: docs/getstarted.md [hunk 1]
```diff
@@ -99,6 +99,11 @@ the chaos to their respective work-namespaces in shared environments.
 - In all subsequent steps, please follow these instructions by replacing the nginx namespace and labels with that of your 
 application.
 
+- The chaos operator collects some usage metrics (operator installation count & experiment run count) via a google analytics
```
Comment (rahulchheda) [inline docs/getstarted.md:102]:
```suggestion
- The chaos runner collects some usage metrics (operator installation count & experiment run count) via a google analytics 
```

Comment (rahulchheda) [reply docs/getstarted.md:102]: Should we change it the component responsible?
cc: @ksatchit

Comment (ksatchit) [reply docs/getstarted.md:102]: operator broadly should do I suppose @rahulchheda . We are not introducing runner anywhere in the getstarted page. And it is, after all the secondary/child resource of operator. Also, the env we set at operator level is propagated to runner internally.

**Review by rahulchheda — APPROVED** [review]

**Review by ajeshbaby — APPROVED** [review]
LGTM, However post the changes in env, does the user required to restart any service if yes that needs to be documented. Else its good to merge.

**Comment by chandankumar4** [comment]
> LGTM, However post the changes in env, does the user required to restart any service if yes that needs to be documented. Else its good to merge.

After changing the ENV the operator will be restarted, And any chaos run after that will not send the analytics data

**Pull request merged**